keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ResourceEvent.kt

```diff
@@ -67,7 +67,7 @@ import org.slf4j.LoggerFactory
 abstract class ResourceEvent(
   open val message: String? = null,
   override val triggeredBy: String? = null
-) : PersistentEvent() {
+) : PersistentEvent(), ResourceHistoryEvent {
```

Comment:
The new `ResourceHistoryEvent` interface provides a common type for events of any scope that belong in the resource event history.